lib/Installation/Partitioner/LibstorageNG/v4_3/AddLogicalVolumePage.pm

```diff
@@ -30,11 +30,12 @@ sub new {
 sub init {
     my $self = shift;
 
-    $self->{tb_lv_name} = $self->{app}->textbox({id => '"Y2Partitioner::Dialogs::LvmLvInfo::NameWidget"'});
-    $self->{rb_system}  = $self->{app}->radiobutton({id => 'system'});
-    $self->{rb_swap}    = $self->{app}->radiobutton({id => 'swap'});
-    $self->{btn_next}   = $self->{app}->button({id => 'next'});
-
+    $self->{tb_lv_name}   = $self->{app}->textbox({id => '"Y2Partitioner::Dialogs::LvmLvInfo::NameWidget"'});
+    $self->{rb_system}    = $self->{app}->radiobutton({id => 'system'});
+    $self->{rb_swap}      = $self->{app}->radiobutton({id => 'swap'});
+    $self->{btn_next}     = $self->{app}->button({id => 'next'});
+    $self->{rb_thin_pool} = $self->{app}->radiobutton({id => 'thin_pool'});
+    $self->{rb_thin}      = $self->{app}->radiobutton({id => 'thin'});
```

Comment:
in the screen appears as a thing volume, could we name it 'rb_thin_volume' ?

Reply:
yes, I thought I should keep consistency with the id because of the rest of the buttons.

Reply:
those ids sometimes have really weird strings, my opinion (and it is just that) is that should be consistent with what we see in the screen.

Reply:
Done